app/model/Image.scala

```diff
@@ -13,23 +13,26 @@ object ImageAssetDimensions {
   def fromThrift(dim: ThriftImageAssetDimensions) = ImageAssetDimensions(dim.height, dim.width)
 }
 
-case class ImageAsset(mimeType: Option[String], file: String, dimensions: Option[ImageAssetDimensions], size: Option[Long]) {
-  def asThrift = ThriftImageAsset(mimeType, file, dimensions.map(_.asThrift), size)
+case class ImageAsset(mimeType: Option[String], file: String, dimensions: Option[ImageAssetDimensions],
+                      size: Option[Long], aspectRatio: Option[String]) {
```

Comment:
Not sure. Currently it is possible to pick something different from the grid by navigating away from the cropping page and selecting a differently sized image. Maybe we shouldn't enforce it on the server since we are not enforcing on the client, only strongly suggesting that these crops should be used.